Razor/UI/Razor.Designer.cs

```diff
@@ -1642,6 +1642,8 @@ private void InitializeComponent()
             this.txtObjDelay.TabIndex = 56;
             this.txtObjDelay.TextAlign = System.Windows.Forms.HorizontalAlignment.Center;
             this.txtObjDelay.TextChanged += new System.EventHandler(this.txtObjDelay_TextChanged);
+            this.txtObjDelay.Text = "500";
```

Comment:
I think we still want this to be configurable because people have different latency. But we should limit the range to between 500 and 800.

Reply:
Ok, i will add validators then and remove ReadOnlyFlag for field